### DISCUSSION TITLE
How to uninstall SDR++ for macOS

### DISCUSSION THREAD

**Discussion by bhall7** [AlexandreRouma/SDRPlusPlus#455] [General]
Hello,

I installed sdrpp_macos_amd64.pkg (1.0.4) today, but after failing to get things running, I just decided to uninstall it for now until things are more mature on macOS.

How can I uninstall SDR++ on macOS? When I issue the command `pkgutil --pkg-info sdrpp` it only shows:
```
package-id: sdrpp
version: 0
volume: /
location: 
install-time: 1634730365
```
Can someone please show the files that the sdrpp_macos_amd64.pkg installer copies on macOS so that I can manually uninstall? Or, are there official uninstall steps for macOS somewhere?

I'm excited for SDR++ to mature further on macOS in the future. Will be eagerly watching the development for macOS--works great on Windows.

Thanks!

**Comment by AlexandreRouma** [comment]
Hello, run these commands:
```sh
sudo rm /usr/local/bin/sdrpp /usr/local/lib/libsdrpp_core.dylib
sudo rm -r /usr/local/lib/sdrpp
sudo rm -r /usr/local/share/sdrpp
sudo rm -r ~/.config/sdrpp
```

**Reply by bhall7** [reply]
Thanks!
